index.sh

```diff
@@ -32,6 +34,59 @@ function remove_images() {
   done
 }
 
+function docker_install_linux(){
+    sudo apt-get install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common -y
+    
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    
+    sudo apt-key fingerprint 0EBFCD88
+    
+    sudo add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu
+    $(lsb_release -cs) \
+    stable"
+    sudo apt-get update
+    sudo apt-get install docker-ce docker-ce-cli containerd.io -y
+    echo " ${GREEN} docker has been installed!"
+}
+
+function docker_install_mac(){
+
+  # Check if Homebrew is installed
+  if command -v brew >/dev/null 2>&1; then
+    echo "Homebrew found, installing Docker using Homebrew..."
+      /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+
+     brew cask install docker
+     echo "${GREEN} docker has been installed!"
+  else
+    echo "Homebrew not found, downloading Docker Desktop..."
+
+    # # Homebrew is not installed, check for Apple Silicon or Intel processor
+
+    if [[ "$(uname -m)" == "arm64" ]]; then
+      curl -o Docker.dmg https://desktop.docker.com/mac/main/arm64/Docker.dmg
+    else 
+      curl -o Docker.dmg https://desktop.docker.com/mac/main/amd64/Docker.dmg
+    fi 
+
+    # Mount the docker dmg file 
+    hdiutil mount Docker.dmg
+    cp -r /Volumes/Docker/Docker.app /Applications/
+    hdiutil unmount /Volumes/Docker
+    rm Docker.dmg   
+  fi
+
+  if [ -x "$(command -v docker)" ]; then
+    echo "${GREEN} docker has been installed!"
+  fi
```

Comment:
In case the docker is ALREADY installed, we should not show this text. 
The issue is that could seem to the user that we installed AGAIN docker, when actually we simply skip.

I tested on my Mac which has Docker already installed and I have this.

```
Fetching https://github.com/vulpemventures/nigiri/releases/latest...
Latest release tag = v0.4.14
Fetching https://github.com/vulpemventures/nigiri/releases/download/v0.4.14/nigiri-darwin-arm64...
Moving binary to /usr/local/bin...
Setting binary permissions...
Checking for Docker and Docker compose...
 docker has been installed!

🍣 Nigiri Bitcoin installed!
```

Reply:
Thanks tiero, I have made this correction. I have also squashed my commit to a single commit for this change. I await further feedbacks and reviews.
Kind regards